src/constraint/uniform/ConstantSampler.cpp

```diff
@@ -0,0 +1,105 @@
+#include <stdexcept>
+#include <aikido/constraint/uniform/ConstantSampler.hpp>
+
+namespace aikido {
+namespace constraint {
+
+//=============================================================================
+class ConstantSamplerSampleGenerator
+  : public constraint::SampleGenerator
```

Comment:
- Rename to `RnConstantSamplerSampleGenerator` (see above).
- Define this class in an anonymous `namespace` since it is only used in this file.